app/webpacker/components/Posts/PostForm.jsx

```diff
@@ -0,0 +1,143 @@
+        tags: formTags.join(','),
+        unstick_at: formIsStickied ? unstickAt : null,
+        sticky: formIsStickied,
+        show_on_homepage: formShowOnHomePage,
+      });
+    } else {
+      createMutation({
+        title: formTitle,
+        body: formBody,
+        tags: formTags.join(','),
+        sticky: formIsStickied,
+        unstick_at: formIsStickied ? unstickAt : null,
+        show_on_homepage: formShowOnHomePage,
+      });
+    }
+  }, [
+    createMutation,
+    editMutation,
+    formBody,
+    formIsStickied,
+    formShowOnHomePage,
+    formTags,
+    formTitle,
+    unstickAt,
+    postId,
+  ]);
+
+  return (
+    <>
+      <Header>
+        {header}
+      </Header>
+      <Form onSubmit={onSubmit}>
+        <FormField>
+          <Form.Input label={I18n.t('activerecord.attributes.post.title')} onChange={setFormTitle} value={formTitle} />
+        </FormField>
+        <FormField>
+          <label>{I18n.t('activerecord.attributes.post.body')}</label>
+          <MarkdownEditor onChange={setFormBody} value={formBody} />
+          {/* i18n-tasks-use t('simple_form.hints.post.body') */}
+          <I18nHTMLTranslate i18nKey="simple_form.hints.post.body" />
+        </FormField>
+        <FormField>
+          <Form.Select label={I18n.t('activerecord.attributes.post.tags')} options={allTags} onChange={(_, data) => { setFormTags(data.value); }} value={formTags} multiple />
+        </FormField>
+        <FormField>
+          <Form.Checkbox label={I18n.t('activerecord.attributes.post.sticky')} onChange={setFormIsStickied} checked={formIsStickied} />
+          { formIsStickied
+            && (
+            <UtcDatePicker
+              placeholderText={I18n.t('activerecord.attributes.post.unstick_at')}
+              isoDate={unstickAt}
+              onChange={(date) => setUnstickAt(date)}
+            />
+            ) }
+        </FormField>
+        <FormField>
+          <Form.Checkbox label={I18n.t('activerecord.attributes.post.show_on_homepage')} onChange={setFormShowOnHomePage} checked={formShowOnHomePage} />
+          <p>{I18n.t('simple_form.hints.post.show_on_homepage')}</p>
+        </FormField>
+        { errors && (
+          <Message negative>
+            <Message.Header>Request Failed:</Message.Header>
+            <Message.List>
+              {(Object.keys(errors).map((err) => (
+                <Message.Item>
+                  {err}
+                  {' '}
+                  {errors[err].join(',')}
+                </Message.Item>
+              )))}
+            </Message.List>
+          </Message>
+        )}
+        { postCreated && !postUpdated && (
+        <Message positive>
+          Post successfully created. View it
+          {' '}
+          <a href={postURL}>here</a>
+        </Message>
+        )}
+        { postUpdated && (
+          <Message positive>
+            Post successfully updated. View it
+            {' '}
+            <a href={postURL}>here</a>
```

Comment:
```suggestion
            <a href={postURL}>here</a>.
```
and same above on line 129.